File: TRBusinessLayer/DataObjects/TaxItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TRBusinessLayer.DataObjects
{
    public class TaxItem
    {
        public string TaxCode { get; set; }
        public decimal TaxBase { get; set; }
        public decimal TaxAmount { get; set; }
        public decimal S4CalcAmount { get; set; }
        public decimal TaxAmtCalculated { get; set; }
        public decimal Adjustments { get; set; }
        public decimal TaxToPay { get; set; }
        public string Note { get; set; }
    }
}

File: TRClient/src/app/Entities/TaxItem.ts
export class TaxItem  {
    TaxCode: string = '';
    TaxBase: number = 0; 
    TaxAmount: number = 0;

    S4CalcAmount: number = 0; 
    Adjustments: number = 0; 
    TaxToPay: number = 0; 
    TaxAmtCalculated: number= 0;
    Note: string = ''; 
    IsOdd: boolean = false;
}